packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -302,20 +302,38 @@ function NetworkReady({ children }: { children: React.ReactNode }) {
 }
 
 function ConnectionFallbackContainer({
+  layout = 'col',
+  imgProps = {
+    className: 'sm:w-[420px]',
+    src: '/images/three-arbinauts.webp',
+    alt: 'Three Arbinauts'
+  },
   children
 }: {
+  layout?: 'row' | 'col'
+  imgProps?: {
+    className?: string
+    src?: string
+    alt?: string
+  }
   children: React.ReactNode
 }) {
   return (
-    <div className="mt-6 flex min-h-[calc(100vh-80px)] flex-col items-center justify-center px-8">
-      {children}
-      <ExternalLink href="https://metamask.io/download">
-        <img
-          className="sm:w-[420px]"
-          src="/images/three-arbinauts.webp"
-          alt="Three Arbinauts"
-        />
-      </ExternalLink>
+    <div className="my-24 flex items-center justify-center px-8">
+      <div
+        className={`flex flex-col items-center md:flex-${layout} md:items-${
+          layout === 'col' ? 'center' : 'start'
+        }`}
+      >
+        {children}
+        <ExternalLink href="https://metamask.io/download">
```

Comment:
when user connected to a wrong network, should we redirect to the metamask download page on click here?

@spsjvc